app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -30,61 +32,67 @@ class ProjectBoard extends React.Component {
     const { createStory, closeHistory, notifications, removeNotification, history } = this.props;
 
     return (
-      <div className="ProjectBoard">
-        <Notifications
-          notifications={notifications}
-          onRemove={removeNotification}
-        />
-
-        <Column title={I18n.t("projects.show.chilly_bin")}
-          renderAction={() =>
-            <AddStoryButton
-              onAdd={() => createStory({
-                state: Story.status.UNSCHEDULED
-              })}
-            />
-          }
-        >
-          <Stories stories={this.props.chillyBinStories} />
-        </Column>
-
-        <Column
-          title={`${I18n.t("projects.show.backlog")} /
-          ${I18n.t("projects.show.in_progress")}`}
-          renderAction={() =>
-            <AddStoryButton
-              onAdd={() => createStory({
-                state: Story.status.UNSTARTED
-              })}
-            />}
-        >
-          <Sprints
-            sprints={this.props.backlogSprints}
+      <DndProvider backend={HTML5Backend}>
+        <div className="ProjectBoard">
+          <Notifications
+            notifications={notifications}
+            onRemove={removeNotification}
           />
-        </Column>
 
-        <Column
-          title={I18n.t("projects.show.done")}
-        >
-          <Sprints
-            sprints={this.props.doneSprints}
-            fetchStories={this.props.fetchPastStories}
-          />
-        </Column>
+          <Column title={I18n.t("projects.show.chilly_bin")}
+            column="chillyBin"
+            renderAction={() =>
+              <AddStoryButton
+                onAdd={() => createStory({
+                  state: Story.status.UNSCHEDULED
+                })}
+              />
+            }
```

Comment:
You could create a new method into this class to avoid re-creating the same function in each render:

```js
class ProjectBoard extends React.Component {
  ...
  onAddStory = () => {
    const { createStory } = this.props;

    return createStory({
      state: Story.status.UNSCHEDULED
    })
  }

  renderColumnAction = () => (
    <AddStoryButton onAdd={this.onAddStory} />
  )
```
Then you call it as `renderAction={this.renderColumnAction}`